src/main/java/sirius/biz/tenants/TenantController.java

```diff
@@ -369,7 +370,14 @@ private String determineOriginalTenantId(WebContext ctx) {
     @Routed("/tenants/select/:1")
     public void selectTenant(final WebContext ctx, String id) {
         if ("main".equals(id) || Strings.areEqual(determineOriginalTenantId(ctx), id)) {
-            auditLog.neutral("AuditLog.switchedToMainTenant").causedByCurrentUser().forCurrentUser().log();
+            UserAccount<?, ?> account = UserContext.getCurrentUser().as(UserAccount.class);
+            auditLog.neutral("AuditLog.switchedToMainTenant")
+                    .hideFromUser()
+                    .causedByUser(account.getUniqueName(), account.getUserAccountData().getLogin().getUsername())
+                    .forUser(account.getUniqueName(), account.getUserAccountData().getLogin().getUsername())
+                    .forTenant(String.valueOf(account.getTenant().getId()),
+                               account.getTenant().getValue().getTenantData().getName())
```

Comment:
use cache -> tenants.fetchCachedTenantName ?

Reply:
the tenant in getValue is already populated from the cache...